src/Process/3dTilesProcessing.js

```diff
@@ -120,21 +117,13 @@ function computeNodeSSE(camera, node) {
 }
 
 export function init3dTilesLayer(context, layer) {
-    if (layer.initialised) {
-        return;
-    }
-    layer.initialised = true;
-
-    Fetcher.json(layer.url, layer.networkOptions).then((tileset) => {
-        const urlPrefix = layer.url.slice(0, layer.url.lastIndexOf('/') + 1);
-        layer.tileIndex = new $3dTilesIndex(tileset, urlPrefix);
-        layer.asset = tileset.asset;
-        requestNewTile(context.view, context.scheduler, layer, tileset.root, undefined).then(
-            (tile) => {
-                layer.object3d.add(tile);
-                tile.updateMatrixWorld();
-                layer.root = tile;
-            });
+    layer.preprocessedPromise.then((layer) => {
```

Comment:
IMHO `init3dTilesLayer` should be removed completely and it's code should move completely in `preprocessLayer`.

And maybe `MainLoop` should be updated to only update layers that have a resolved `preprocessedPromise`.